tests/e2e/forms/forms-against-muted-contacts.spec.js

```diff
@@ -66,7 +66,7 @@ describe('Submitting forms for muted contacts', () => {
     await utils.updateSettings(settings);
   });
 
-  afterEach(() => utils.revertDb());
+  afterEach(async () => { await utils.revertDb(); });
```

Comment:
Nitpick - I think we can get rid of the {} and the semi colon as in 
```suggestion
  afterEach(async () => await utils.revertDb());
```